loris/loris/Dockerfile

```diff
@@ -1,16 +1,29 @@
 FROM ubuntu:16.04
```

Comment:
You could perhaps build this from https://github.com/wellcometrust/dockerfiles/tree/master/python3?

Reply:
We ran Loris on Alpine originally, and saw a small improvement in response times when we switched to Ubuntu (I'm not sure if it was statistically significant tho!).

Might be worth another try at some point, with proper testing.